instrumentation/apache-httpclient/apache-httpclient-4.0/src/main/java/io/opentelemetry/auto/instrumentation/apachehttpclient/v4_0/ApacheHttpClientTracer.java

```diff
@@ -59,8 +56,24 @@ protected String responseHeader(HttpResponse response, String name) {
     return header(response, name);
   }
 
+  @Override
+  protected Setter<HttpUriRequest> getSetter() {
+    return SETTER;
+  }
+
   private static String header(HttpMessage message, String name) {
     Header header = message.getFirstHeader(name);
     return header != null ? header.getValue() : null;
   }
+
+  @Override
+  protected String getInstrumentationName() {
+    return "io.opentelemetry.auto.spring-webflux-5.0";
```

Comment:
This name is wrong

Reply:
FYI this is not a new name.  It exists like this before.  what should the new name be if you think it's not right?

Reply:
This is `ApacheHttpClientTracer`. So name could be `io.opentelemetry.auto.apache-http-client`